cmake/modules/GoogleBenchmark.cmake

```diff
@@ -19,7 +19,7 @@ ExternalProject_Add(
   #            -Dgtest_force_shared_crt=ON
   CMAKE_ARGS -G ${CMAKE_GENERATOR}
   -DCMAKE_INSTALL_PREFIX:PATH=${GBENCHMARK_PREFIX}
-  -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+  -DCMAKE_BUILD_TYPE:STRING=RELEASE
```

Comment:
Wouldn't it be better in  the long run to have a BENCHLIB_CMAKE_BUILD_TYPE that default to Release just in case we have the need to debug the benchmarkmark library itself?

Reply:
Yes, you are right! I closing this PR, and opening a new one